utils.go

```diff
@@ -113,10 +116,28 @@ func OpenMainMenu() {
 	C.OpenMainMenu()
 }
 
-func LoadKeyboard() {
-	C.LoadKeyboard(C.CString(defaultKeyboardLang))
-}
+func GetConfig() (map[string]string, error) {
+	file, err := os.Open(C.GLOBALCONFIGFILE)
+	if err != nil {
+		return nil, err
+	}
+	defer file.Close()
+
+	configs := make(map[string]string)
+	scanner := bufio.NewScanner(file)
+	for scanner.Scan() {
+		line := scanner.Text()
+		if strings.Contains(line, "=") {
+			parts := strings.Split(line, "=")
+			if len(parts) == 2 {
+				configs[parts[0]] = parts[1]
+			}
+		}
```

Comment:
Probably doesn't matter that much, but you can directly call `strings.SplitN(line, 2)` and check that it indeed split into 2 parts. `Contains` does the search the second time, which can be avoided.